v4/plugins/eclipse/che-machine-exec/nightly/devfile.yaml

```diff
@@ -0,0 +1,25 @@
+schemaVersion: "2.0.0"
+metadata:
+  name: "Che Machine Exec"
+components:
+- name: terminal
+  attributes:
+    "app.kubernetes.io/name": che-terminal.eclipse.org
+    "app.kubernetes.io/part-of": che.eclipse.org
+    "app.kubernetes.io/component": terminal
+  container:
+    image: "quay.io/eclipse/che-machine-exec:nightly"
+    command: ['/go/bin/che-machine-exec']
+    args:
+      - '--url'
+      - '0.0.0.0:4444'
+      - '--pod-selector'
+      - controller.devfile.io/workspace_id=$(DEVWORKSPACE_ID)
+    endpoints:
+      - name: "che-mach-exec"
+        exposure: public
+        targetPort: 4444
+        protocol: ws
+        secure: false
```

Comment:
Note that it's marked as unsecure by purpose, since it's not implemented yet on Che side for DevWorkspace

And it listen to 0.0.0.0 which is again different from v3 due the same reason, DevWorkspace does not have proxy that would expose localhost to 0.0.0.0 in a secure manner.